src/snowflake/snowpark/dataframe.py

```diff
@@ -3374,7 +3372,7 @@ def _explain_string(self) -> str:
             if exec_plan:
                 msg = f"{msg}\nLogical Execution Plan:\n{exec_plan}"
             else:
-                msg = f"{self._plan.queries[0].sql} can't be explained"
+                msg = f"{self._plan.queries[0].sql} can't be explained"  # pragma: no cover
```

Comment:
I wonder when this would happen

Reply:
@sfc-gh-jdu Do you know what sql can't be explained?